tool/plugin/lib/plugin.dart

```diff
@@ -81,14 +90,28 @@ abstract class ProductCommand extends Command {
 
   bool get isForIntelliJ => argResults['ij'];
 
-  String get release => globalResults['release'];
+  String get release {
+    var rel = globalResults['release'];
+    if (rel != null && rel.startsWith('=')) {
```

Comment:
This should come in from the args package w/o the leading `=`.

Reply:
True, but I can't keep track of when I need '=' and when I don't. If I type "-r=19" then there's an additional '=' to be removed. I dislike programs that are too unnecessarily strict in parsing options. I can remove it you want the strict behavior.

Reply:
No, I'm just surprised that this isn't somehow already handled by `package:args`; sounds like it's not -